### PR TITLE
Add Brazil Document Generators

### PR DESCRIPTION
**Pull request by eltonsandre**
- CPF: natural person identification document generator
- CNPJ: legal entity identification document generator

### CODE REVIEW

**Comment by eltonsandre** [comment]
As this repository already has a long period without maintenance and addition of new Fakers, the same code was merged into the datafaker repository, which offers the same functionality as Faker:
https://github.com/datafaker-net/datafaker/

File: src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    public static String cnpj(boolean formatted, boolean valid) {
+        String cnpj = "";
+
+        if (valid) {
+            StringBuilder partial = new StringBuilder();
+            for (int i = 0; i < 8; i++) {
+                partial.append((int) (Math.random() * 9));
+            }
+
+            cnpj = partial.append("0001").toString();
```
Comment (alesaudate) [inline src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: Why 0001 always ? That would assume all companies only have one branch - which, of course, is not true.

Comment (bodiam) [reply src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: What should the logic be @alesaudate ? I'm maintaining an up to date fork of this library (Datafaker.net), and this code is part of it. Happy to fix it in my fork, I'm planning on making a 1.2 release in several days

Comment (alesaudate) [reply src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: HI @bodiam , I have just seen the fork. It seems it makes more sense to do changes over there, so I'm thinking about contributing to datafaker myself. Thanks for the reply :)

Comment (alesaudate) [reply src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: The change itself should be that this number should be random also. Its position on a CNPJ refers to the company branch, so 0001 would always point to the first branch. It's the most common kind of CNPJ, given that several companies only have one branch anyway, but it's not the only possibility. McDonald's, for example, has hundreds of branches in Brazil (at least it's main local subsidiary has, I don't know how many McDonald's subsidiaries are there in Brazil).

Comment (bodiam) [reply src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: @alesaudate  your contributions would be very welcome, thank you! If you would like to make a change to this logic to make it more real, that would be a very welcome contribution!

Comment (eltonsandre) [reply src/main/java/com/github/javafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java:27]: @alesaudate 
Initially to reduce complexity, mock was just for parent company. But we can add a parameter to generate documents for branches.

**Pull request closed without merging**